mm2src/coins_activation/src/l2/init_l2.rs

```diff
@@ -199,10 +204,15 @@ where
         )
         .await?;
 
-        lp_register_coin(&self.ctx, coin.into(), RegisterCoinParams {
-            ticker: self.ticker.clone(),
-        })
-        .await?;
+        if let Ok(c_ctx) = CoinsContext::from_ctx(&self.ctx) {
```

Comment:
We must return an error in this case. You can add `From<CryptoCtxError>` into this:
https://github.com/KomodoPlatform/atomicDEX-API/blob/42c84d0ec92d74b4a4085ee5fe62b4f2360a56f6/mm2src/coins_activation/src/l2/init_l2.rs#L42